apps/web/src/services/dashboards/shared/dashboard-widget-container/DashboardWidgetContainer.vue

```diff
@@ -191,6 +189,39 @@ onMounted(async () => {
 
 </script>
 
+<template>
+    <div ref="containerRef"
+         class="dashboard-widget-container"
+    >
+        <template v-if="!dashboardDetailState.loadingDashboard">
+            <template v-for="(widget) in reformedWidgetInfoList">
+                <component :is="widget.component"
+                           :id="widget.widget_key"
+                           :key="widget.widget_key"
```

Comment:
nitpick: In `vue3`, `<template>` must have `key`, not child component.
```suggestion
            <template v-for="(widget) in reformedWidgetInfoList"
                              :key="widget.widget_key"
            >
                <component :is="widget.component"
                           :id="widget.widget_key"
```